ClientApp/viking-vault/src/components/UserPage.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import SideBar from './SideBar'
+import TopBar from './TopBar'
+import AccountPage from './AccountPage'
+import UserIcon from './UserIcon';
+
+interface IUserPageState {
+    type:boolean;
+}
+class UserPage extends React.Component<any, IUserPageState>{
+    constructor(props:any){
+        super(props)
+
+        this.state = {
+            type: true
+        } 
+        this.callbackFunction = this.callbackFunction.bind(this);
+    }
+
+    callbackFunction = (childData: boolean) =>{
```

Comment:
From the name of the parameter you'd expect that `childData` holds something more complex, like an object, in here it's only a boolean value, could you name to something more suggestive to explain what it represent by its naming?